server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -12,6 +13,7 @@ export async function runWithRetry<T>(
     maxRetries: number,
     retryAfterMs: number,
     logger?: ILogger,
+    lumberjack?: typeof Lumberjack,
```

Comment:
Why do we have to make Lumberjack as a parameter?
Can we just reference the global Lumberjack instead?